runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -458,6 +452,31 @@ public void onChatMessage(ChatMessage event)
 		{
 			removeGameTimer(STAFF_OF_THE_DEAD);
 		}
+
+		if (config.showFreezes() && event.getMessage().equals("<col=ef1020>You have been frozen!</col>"))
+		{
+			createGameTimer(ICERUSH, false);
```

Comment:
So this will create 4 different timers? That seems weird

Reply:
Please read the PR

Reply:
I have to agree with db here... I'd prefer a solution that assumes a freeze is of one type (Ice Barrage would probably be best) and corrects itself should a graphic of another type appears.

Reply:
@Nightfirecat @deathbeam Freeze message defaults to barrage. If graphic appears, barrage timer is removed and correct freeze time is created. If no graphic appears, default timer is kept (barrage).